### PR TITLE
feat(monitoring): make engine.buckets.backlog per worker

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: Ief6665d27f161f701084ade646684d6dc4c87b40

### CODE REVIEW

**Comment by sileht** [comment]
This pull request is part of a stack:
1. feat(monitoring): make engine.buckets.backlog per worker ([#3656](https://github.com/Mergifyio/mergify-engine/pull/3656))
1. feat(monitoring): add engine.buckets.events.count ([#3657](https://github.com/Mergifyio/mergify-engine/pull/3657))

File: mergify_engine/worker.py [hunk 1]
```diff
@@ -1201,11 +1231,12 @@ async def async_status() -> None:
     worker_count: int = worker_per_process * process_count
 
     redis_stream = utils.create_yaaredis_for_stream()
-    org_bucket_selector = SharedOrgBucketSelector(redis_stream, 0, worker_count)
 
     def sorter(item: typing.Tuple[bytes, float]) -> int:
         org_bucket, score = item
-        return org_bucket_selector.get_shared_worker_id_for(org_bucket)
+        return SharedOrgBucketSelector.get_shared_worker_id_for(
+            org_bucket, worker_count
```
Comment (jd) [inline mergify_engine/worker.py:1238]: I'm confused, what is worker_count here? where is it defined?

Comment (sileht) [reply mergify_engine/worker.py:1238]: L1231

Comment (jd) [reply mergify_engine/worker.py:1238]: Ok I missed this was a closure 🤦

**Review by jd — APPROVED** [review]

**Pull request merged**